internal/driver.go

```diff
@@ -77,8 +79,7 @@ func NewRegistryDefaultWithDSN(t testing.TB, dsn string) (*config.Config, *drive
 	ctx := context.Background()
 	c := NewConfigurationWithDefaults(t)
 	c.MustSet(ctx, config.ViperKeyDSN, stringsx.Coalesce(dsn, dbal.NewSQLiteTestDatabase(t)))
-
-	reg, err := driver.NewRegistryFromDSN(ctx, c, logrusx.New("", ""))
+	reg, err := driver.NewRegistryFromDSN(ctx, c, logrusx.New("", "", logrusx.ForceLevel(logrus.ErrorLevel)))
```

Comment:
With this, the migration logs are hidden, correct? I guess this replaces https://github.com/ory/kratos/pull/3562 then?

Reply:
#3562 is kinda dangerous IMO, because it competely drops logs including error logs. This change silences warn+info logs only during testing.

Reply:
Feel free to revert in this PR then, as it's already merged. Sorry.

We do the same thing as #3562 in cloud, so I pretty much copied it over from there.

Reply:
#3562 also only drops the logs in testing, though.